docs/GettingStarted.rst

```diff
@@ -1289,6 +1289,51 @@ You may display the home page to verify that the news is available in the approp
 |image172|
 
 
+Unpinning a News in the home page
+----------------------------------
+
+As a platform-wide publisher (publisher:/platform/users role required), you are allowed to unpin any pinned article from the home page. 
```

Comment:
The required role is publisher:/platform/web-contributors.

Reply:
changed for pin and unpin